oci.sh

```diff
@@ -47,7 +47,7 @@ for i in $INSTANCES; do
         continue
     fi
 
-    if [ $DRY_RUN == "true" ]; then
+    if [ "$DRY_RUN" == 'true' ]; then
```

Comment:
+1 for adding double quotes around the `$DRY_RUN` variable. 

However why change the double quotes around `true` with single quotes ? 

This also deviates from the pattern used elsewhere in the source code, for example here:
https://github.com/osbuild/cloud-cleaner/pull/31/files#diff-13c5717db0bbbc3eb9e1afbb8113d2bcc03a7aab9dedb451e2d35b93ebe91fdfR74